README.md

```diff
@@ -1 +1,183 @@
-# Budget-app
\ No newline at end of file
+# Budget-App
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget-App ](#-rails-Budget-App-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Dependencies](#dependencies)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget-App <a name="about-project"></a>
+
+**Budget-App** The "Budget App," is a dynamic and user-friendly web application designed to empower individuals to take control of their finances. With a focus on effective budget management, this project offers a seamless platform for users to track, categorize, and analyze their transactions, providing valuable insights into their spending patterns.
+
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://budgetapp-7kxz.onrender.com/home)
```

Comment:
- [ ] Good job on deploying your website. However, you need to also add the link to your video presentation as per the requirement. Please add it.